base/bin/migration.mustache

```diff
@@ -2,9 +2,9 @@ const fs = require('fs');
 const mustache = require('mustache');
 
 const now = new Date().toISOString();
-const migrationName = now + '_' + process.argv[2];
+const migrationName = `${now}_${process.argv[2]}`;
```

Comment:
Now that you mention it, using `let` is only available in node 6. `var` has a lint error though - should we disable the rule?